src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -752,8 +767,64 @@ private static Expression BindParameterFromRouteValueOrQueryString(ParameterInfo
             return BindParameterFromValue(parameter, Expression.Coalesce(routeValue, queryValue), factoryContext);
         }
 
-        private static Expression BindParameterFromFormFile(ParameterInfo parameter, MemberExpression property, string key, FactoryContext factoryContext) =>
-            BindParameterFromValue(parameter, GetValueFromProperty(property, key, typeof(IFormFile)), factoryContext);
+        private static Expression BindParameterFromFormFile(ParameterInfo parameter, string key, FactoryContext factoryContext)
```

Comment:
I see what you mean, it is 90% shared...

Reply:
Once #35167 is merged and I fix-up any conflicts, I'll try factoring it out into a separate method that `BindParameterFromValue()` and `BindParameterFromFormFile()` can both use.

Reply:
be491e0486e82e60eaebf559f48e31686f37c245